### PR TITLE
IB known issue update on federated users

### PR DESCRIPTION
**Pull request by anwara-ms**
IB known issue update on federated users. IB doesnt work on federated users.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -197,6 +197,8 @@ For more information on licenses and permissions, including plans and pricing, s
 
 - **Users can't join channel meetings**: If IB policies are enabled, users aren't allowed to join channel meetings if they're not a member of the team. The root cause is that IB checks rely on whether users can be added to a meeting chat roster, and only when they can be added to the roster are they allowed to join the meeting. The chat thread in a channel meeting is available to Team/Channel members only, and non-members can't see or access the chat thread. If IB is enabled for the tenant and a non-team member attempts to join a channel meeting, that user isn't allowed to join the meeting. However, if IB is _not_ enabled for the tenant and a non-team member attempts to join a channel meeting, the user is allowed to join the meeting—but they won't see the chat option in the meeting.
 
+- **IB doesnt work for federated users**: If you allow federation with external tenants, then the users of those tenants wont be restricted by IB policies. Also, if users of your organization join a chat or meeting organized by external federated users, then IB policies wont restrict communication between users of your organization.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:200]:
```suggestion
- **IB policies don't work for federated users**: If you allow federation with external tenants, the users of those tenants won't be restricted by IB policies. If users of your organization join a chat or meeting organized by external federated users, then IB policies also won't restrict communication between users of your organization.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@chrfox PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by anwara-ms** [comment]
@chrfox @robmazz - can you please go live with this change? I see there is a conflict, I dont know how to resolve it.

**Comment by anwara-ms** [comment]
Rob/Chris- Following up. Can you plz merge and publish?

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

**Comment by robmazz** [comment]
@anwara-ms  - yes, merging now.

**Comment by anwara-ms** [comment]
Cool. Thanks.

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

**Pull request merged**